src/universal/utils/auth0Login.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Sends an authentication request to auth0 for the client app.
+ *
+ * When the request is successful a redirect occurs.
+ * When it fails, the promise rejects.
+ *
+ * @flow
+ */
+
+import type {Credentials} from 'universal/types/auth';
+import WebAuth from 'auth0-js';
+
+export default function auth0Login(webAuth: WebAuth, credentials: Credentials): Promise<void> {
+  return new Promise((resolve, reject) => {
+    webAuth.login({
```

Comment:
id still love to see this promisified

Reply:
Tried to do it, but the lib did not work for me!